app/presenters/embassy_presenter.rb

```diff
@@ -4,27 +4,17 @@ class EmbassyPresenter < SimpleDelegator
   include ActionView::Helpers::TagHelper
   include ActionView::Helpers::UrlHelper
 
-  def remote_office_path
-    if special_case?
-      SPECIAL_CASES[name][:base_path]
-    elsif has_remote_service?
-      organisation.public_path
-    end
-  end
-
-  def remote_office_name
-    if special_case?
-      SPECIAL_CASES[name][:building]
-    elsif has_remote_service?
-      organisation.name
-    end
-  end
+  RemoteOffice = Struct.new(:name, :location, :path)
```

Comment:
Minor: It might be nice to use the `keyword_init` option so that constructor call-sites have to use keyword arguments which might make things clearer, i.e.

```ruby
RemoteOffice = Struct.new(:name, :location, :path, keyword_init: true)

ro = RemoteOffice.new(name: 'name', location: 'location', path: 'path)
```

Reply:
Resolved this by editing the history. Force pushing with the changes.